src/libs/Navigation/FreezeWrapper.tsx

```diff
@@ -27,11 +28,9 @@ function FreezeWrapper({keepVisible = false, children}: FreezeWrapperProps) {
             // if the screen is more than 1 screen away from the current screen, freeze it,
             // we don't want to freeze the screen if it's the previous screen because the freeze placeholder
             // would be visible at the beginning of the back animation then
-            if ((navigation.getState()?.index ?? 0) - (screenIndexRef.current ?? 0) > 1) {
-                setIsScreenBlurred(true);
-            } else {
-                setIsScreenBlurred(false);
-            }
+            const navigationIndex = (navigation.getState()?.index ?? 0) - (screenIndexRef.current ?? 0);
+            const shouldSetScreenBlurred = Platform.OS === 'android' || Platform.OS === 'ios' ? navigationIndex > 1 : navigationIndex >= 1;
```

Comment:
Our code style say we should not use platform checks like this, rather we should add the platform specific code to its specific file as you can see in bunch of other places in the app. Could you please update that? Thanks!

Reply:
Should we have a separate utils file like `Navigation/utils/screenBlurredProperties.tsx` and `Navigation/utils/screenBlurredProperties.native.tsx`? That way we can have a specific function to return this and follow the guidelines

Reply:
Yep I think we should move it to separate files, hopefully there will be nice way to do that without too much code repetition

Reply:
@danieldoglas That's a great suggestion. I would try to incorporate this tomorrow morning 👍